### DISCUSSION TITLE
Regarding making themes under Greener Coding project

### DISCUSSION THREAD

**Discussion by SAMAD101** [wagtail/gsoc#74] [Q&A] [answered]
In the [Greener coding: Wagtail's climatic impact](https://github.com/wagtail/gsoc/blob/main/project-ideas.md) we also have to make themes for the project. I wanted to know if the themes will only be for the admin interface?

**Comment by laymonage** [comment, marked as answer]
For the most part, yes. I think the only place outside the admin where we'd like to apply the themes would be the userbar:

<img src="https://user-images.githubusercontent.com/51043550/214340572-1a949abd-0fd1-4478-82c7-2922840a6dae.png"/>

**Reply by SAMAD101** [reply]
Makes sense. Thanks for clearing that up.

**Comment by thibaudcolas** [comment]
Sage’s answer is spot on. I’ve reworded the document to make it clearer the themes are just one of many changes we could introduce, and added another example.
